pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SurveyTrigger.java

```diff
@@ -60,4 +65,37 @@ public static Map<String, SurveyTrigger> getSurveyTriggers() {
         logger.info("Found " + surveyTriggers.size() + " survey triggers ");
         return surveyTriggers;
     }
+
+    public static long insertTrigger(@NonNull String userId, @NonNull String reason, long currentTime) {
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
+            try (PreparedStatement stmt = conn.prepareStatement(
+                    DSMConfig.getSqlFromConfig(ApplicationConfigConstants.INSERT_SURVEY_TRIGGER), Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
You are not tripping the linter with this long line?

Reply:
I don't appear to be.  My IDE isn't complaining about it.

Reply:
I'll move it to two lines.